bulbs/special_coverage/templates/special_coverage/bulbs_sc_landing.html

```diff
@@ -59,12 +59,22 @@
         {% endif %}
         <bulbs-video-meta
           src='{{ VIDEOHUB_BASE_URL }}/video/{{current_video}}.json'
-          share-email-message="{{share_message}}"
-          share-twitter-handle="{{twitter_handle}}"
           data-track-category="Special Coverage Landing"
-          share-track-action="Video: Main: Share"
         >
         </bulbs-video-meta>
+        <share-tools
+          class="video-carousel-share-tools"
+          share-url={% build_video_share_uri current_video %}
+          share-title={{current_video.title}}
+          share-email-message="{{share_message}}"
+          share-twitter-handle="{{twitter_handle}}"
+          share-track-action="Video: Main: Share">
```

Comment:
A bit nitpicky, but for these attributes, can we tab them in another tab? Visually separates the element and it's attributes from the nested content.